### DISCUSSION TITLE
Manually call openConnectModal outside of <ConnectButton.Custom>?

### DISCUSSION THREAD

**Discussion by Sam-Scolari** [rainbow-me/rainbowkit#432] [Q&A] [answered]
I have a feature where people can upvote something but they need to be connected before they can use that feature. Is there a way I can call openConnectModal within the onClick function of my upvote button if they aren't connected?

**Comment by markdalgleish** [comment, marked as answer]
This is out now in [RainbowKit v0.4.3.](https://github.com/rainbow-me/rainbowkit/releases/tag/%40rainbow-me%2Frainbowkit%400.4.3)

You can check out the docs here: https://www.rainbowkit.com/docs/modal-hooks
